libparsec/crates/client_connection/tests/proxy/mod.rs

```diff
@@ -0,0 +1,169 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
```

Comment:
Avoid folder with single `mod.rs`, it's more redable to just have `proxy.rs` instead

Reply:
I would need to disable `autotest`, that would work ?

Reply:
Ha ! you're right !

In that case just leave this as-is ;-)

(also note in the future we will most likely move all tests in `tests/unit` which will, among other things, allow us to rename this correctly ^^)